tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUtilities.java

```diff
@@ -599,19 +602,20 @@ public void TestResolvedVoteCounts() {
         resolver.clear();
         resolver.setBaileyValue("bailey");
         resolver.setLocale(CLDRLocale.getInstance("de"), null);
-        resolver.setTrunk("foo", Status.approved);
+        resolver.setBaseline("foo", Status.approved);
         resolver.add("zebra", toVoterId("googleS"));
         counts = resolver.getResolvedVoteCounts();
         logln(counts.toString());
         assertEquals("", "foo", new ArrayList<>(counts.keySet()).get(0));
     }
 
     private void verifyRequiredVotes(VoteResolver<String> resolver, String locale,
-        String xpath, int required) {
+        String xpath, Status baselineStatus, int required) {
```

Comment:
This is the essential change for testing; previously the baseline status often defaulted to missing. Now we test with various status values.